### PR TITLE
Update external-ids-upsert-block

### PR DESCRIPTION
**Pull request by amaster507**
Changed the example to a more generalized use case with actual external identifiers from a 3rd party source (imdb).

Added an example of correcting an external identifier.

Added a few notes.

Added an advanced section in regards to mapping GraphQL <--> DQL Schemas in regards to external identifiers.

### CODE REVIEW

**Comment by skrdgraph** [comment]
@amaster507 could you please complete the license/cla?

File: content/mutations/external-ids-upsert-block.md [hunk 1]
```diff
@@ -6,81 +6,160 @@ weight = 4
...
+}
 ```
+
+Create initial data of a movie and an actor with external identifiers (example uses imdb identifiers).
+```DQL
 {
   set {
-    _:blank <xid> "http://schema.org/Person" .
-    _:blank <dgraph.type> "ExternalType" .
+    _:nm0000705 <dgraph.type> "Actor" .
```
Comment (MichelDiz) [inline content/mutations/external-ids-upsert-block.md:37]: We should keep using "Blank" in docs. Always doing this to unconsciously maintain that it is a Blank node. For the person to get used to the terminology.

**Review by MichelDiz — COMMENTED** [review]
This change isn't good. Either you keep what we had as it is or separate the two examples. I see that your idea is to talk about the relationship of external IDs with GraphQL (it seems). But the original example was contextualizing with W3C standard external IDs. We shouldn't drop this.

Keep what you want to do as something extra. Not a replacement.

Cheers.

**Comment by amaster507** [comment]
No problem, I'll just leave it in discuss then and users can find it there. I'll close this PR as it is unwanted. Thank you.

**Comment by MichelDiz** [comment]
It is not unwanted Anthony, any new information is good to document. But other/old stuff should be around. GraphQL and W3C users are different public.

**Pull request closed without merging**